draft-ietf-acme-acme.md

```diff
@@ -1801,6 +1802,39 @@ If all of the above verifications succeed, then the validation is successful.
 If no DNS record is found, or DNS record and response payload do not pass these
 checks, then the validation fails.
 
+# Use with CAA Records
+
+An ACME server SHOULD support CAA DNS records as described in {{RFC6844}}. The
+server SHOULD look for such records when issuing authorizations, as opposed to
+when issuing certificates.
```

Comment:
Since authorizations may last a long time, maybe the implementing CA should check at both authorization time and (eventual) issuance time.